tools/functional-tester/etcd-runner/main.go

```diff
@@ -212,6 +215,170 @@ func runRacer(eps []string, round int) {
 	doRounds(rcs, round)
 }
 
+func runWatcher(eps []string) {
+	// watcher for setting watch on prefixes and also handling the response on watches
+	type watcher struct {
+		prefix string
+		c      *clientv3.Client
+		wc     clientv3.WatchChan
+	}
+
+	var runningTime time.Duration = 60 // time for which operation should be performed
+	noOfPrefixes := 36                 // total number of prefixes which will be watched upon
+	watchPerPrefix := 10               // number of watchers per prefix
+	reqRate := 20                      // put request per second
+	maxKeys := 100000                  // max number of keys for put operation
```

Comment:
change this to keyPrePrefix. see https://github.com/coreos/etcd/pull/6635/files#r83473830